requirements.txt

```diff
@@ -0,0 +1,5 @@
+# DO NOT include azure-functions-worker in this file
+# The Python Worker is managed by Azure Functions platform
+# Manually managing azure-functions-worker may cause unexpected issues
+
+azure-functions
```

Comment:
please, move this file to solutions/wordcloud_azure_function
make sure that this file contains all libraries to run Azure function and generate wordcloud